client/auth/login.jsx

```diff
@@ -103,27 +101,58 @@ module.exports = React.createClass( {
 	},
 
 	canSubmitForm: function() {
+		const auth = this.state.auth;
```

Comment:
This isn't wrong by any measure, but you are also free to use destructuring, which I think I see more often in our codebase.

```js
const { auth } = this.state;
```